TWLight/users/models.py

```diff
@@ -158,8 +158,8 @@ class Meta:
     affiliation = models.CharField(max_length=128, blank=True)
 
     def encode_wp_username(self, username):
-        result = urllib.quote(username)
-        return result
+        _ = username.encode('utf-8')
```

Comment:
Just a stylistic thing - `_` is generally used as a variable name when that variable isn't used anywhere, but we do use this below, so it should be named for readability.